data-model/developer-schema/annotation/0.2.0/schema/annotation-for-mas.json

```diff
@@ -0,0 +1,335 @@
+          "examples": [
+            "https://hdl.handle.net/20.5000.1025/XXX-XXX-XXX",
+            "https://doi.org/10.22/XXX-XXX-XXX"
+          ]
+        },
+        "@type": {
+          "type": "string",
+          "description": "This is the handle to the type of the target object.",
+          "pattern": "https://hdl\\.handle\\.net/\\w+/(.){3}-(.){3}-(.){3}",
+          "examples": [
+            "https://doi.org/21.T11148/bbad8c4e101e8af01115",
+            "https://doi.org/21.T11148/894b1e6cad57e921764e"
+          ]
+        },
+        "ods:ID": {
+          "type": "string",
+          "description": "This is the PID of the target object. Valid targets are the Digital Specimen, Digital Media Object or another annotation.",
+          "examples": [
+            "https://hdl.handle.net/20.5000.1025/XXX-XXX-XXX",
+            "https://doi.org/10.22/XXX-XXX-XXX"
+          ]
+        },
+        "ods:type": {
+          "type": "string",
+          "description": "This is the handle to the type of the target object.",
+          "pattern": "https://hdl\\.handle\\.net/\\w+/(.){3}-(.){3}-(.){3}",
+          "examples": [
+            "https://doi.org/21.T11148/bbad8c4e101e8af01115",
+            "https://doi.org/21.T11148/894b1e6cad57e921764e"
+          ]
+        },
+        "oa:hasSelector": {
+          "type": "object",
+          "description": "Optional field to indicate the part of the target object that is being annotated. It can be a field, a class or a region of interest.",
+          "oneOf": [
+            {
+              "type": "object",
+              "properties": {
+                "@type": {
+                  "type": "string",
+                  "const": "ods:FieldSelector",
+                  "description": "A selector for an individual field."
+                },
+                "ods:field": {
+                  "type": "string",
+                  "description": "The full jsonPath of the field being annotated. Following: https://goessner.net/articles/JsonPath/index.html#e2",
+                  "examples": [
+                    "$.occurrences[0].location.dwc:country"
+                  ]
+                }
+              },
+              "required": [
+                "@type",
+                "ods:field"
+              ],
+              "additionalProperties": false
+            },
+            {
+              "type": "object",
+              "properties": {
+                "@type": {
+                  "type": "string",
+                  "const": "ods:ClassSelector",
+                  "description": "A selector for an individual class."
+                },
+                "ods:class": {
+                  "type": "string",
+                  "description": "The full jsonPath of the class being annotated. Following: https://goessner.net/articles/JsonPath/index.html#e2",
+                  "examples": [
+                    "$.occurrences[0].location.georeference"
```

Comment:
Also update this example since occurrences is not in the data model anymore

Reply:
Fixed